### ISSUE TITLE
Cant connect after accepting a game 

### ISSUE THREAD

**Issue by dogfacetanker1** [ValveSoftware/Dota2-Gameplay#15491]
### Description

After I accept to play the game I can not move into character selection. I automatically get punished for an abandon. This happened until I had a 45 min ban. Now it says can not Olay while running as an administrator. All this happened after the new update. Please help. 

### Example Match ID (and possibly Timestamp)

922151825 1552

**Comment by dadobt** [comment]
same , usually when we have party from eu + us players !

**Comment by Lemonlemmings** [comment]
Hi! Have you tried the methods listed here? - https://github.com/ValveSoftware/Dota2-Gameplay/issues/15339#issuecomment-1936596464 - I recommend giving those a go since they have solved the problem for other players!

**Comment by dogfacetanker1** [comment]
Yes, I have tried all of that. The only thing I've found is that as soon as I start the game I can start a game. But as soon as I accept it will no go to character screen and then to the same problem again. Can't run as an administrator.

**Comment by TornardoThorsten** [comment]
I tried all of that as well not working from me... :(

**Comment by dogfacetanker1** [comment]
I'm in contact with steam. They're giving me all the same advice. The most success I've had so far was deleting the bun file and verifying integrity. But still can't connect. Just got another abandon and a 14 min penalty.

**Comment by Riismannen** [comment]
This is getting quite ridiculous, I’ve been able to sometimes get it to work, but most of the time, abandon with long wait penalties, this is me following all suggestions, can we please get some better assistance so we can get to play the game we love #valvegetonit

**Comment by Madjed00** [comment]
same issue and Valve dont care

**Comment by dogfacetanker1** [comment]
I deleted the bin file again, verified, then restarted computer. I'm about to play mid.

**Comment by dogfacetanker1** [comment]
I am in game.

**Comment by Riismannen** [comment]
I just reinstalled the game, deleted bin, verified files and again kicked out, now for an hour.

**Comment by Lemonlemmings** [comment]
@Riismannen - As is recommended in the comment I linked, to avoid being marked as not connecting to games always try spectating a match first. If you can successfully spectate a match, then you _should_ successfully queue.

See this comment on some additional tricks you can try - https://github.com/ValveSoftware/Dota2-Gameplay/issues/15339#issuecomment-1946912658 - If those fail, then it may be worth taking a look into the console log and posting it so that others can help better diagnose your issue. Check for any identifying data before you do post anything!

**Comment by TornardoThorsten** [comment]
I tried it this morning was able to successfully watch one game and not its not working anymore. 
I reinstalled steam, tried the bin deleting workaround but nothing seems to help

**Comment by Riismannen** [comment]
@Lemonlemmings 
Thank you for your feedback, I’m able to watch live games and replays, still not able to connect when I find a match. All the other tips forwarded I’ve also tried. How do I look at my console log?

**Comment by Riismannen** [comment]
@Lemonlemmings 
I specifically tried to join an east US game server and succeeded, could it be an issue going to other regions I saw recently another post of someone having difficulties connecting to other regions

**Comment by Lemonlemmings** [comment]
@Riismannen - Thanks for your feedback. I'll update the original comment with the additional information you've provided.

**Comment by Bbesnik5** [comment]
Ive been having same issue all day today, keep getting timeouts for up to 2 hours bcz of this ... playing with friends in EU It wont let me load on game so I cant even get match ID...

**Comment by Riismannen** [comment]
@Lemonlemmings @Bbesnik5 
So I’ve been playing fine now. Going on my own region US East. I’m attempting us west today as well. Hard part is that I like to game early am and takes long to find games, anyways, this is huge improvement and something valve hopefully is trying to fix.

**Comment by acidlhouse** [comment]
I have the same issue been like this for a week now ive un-installed re-installed nothing is working, try a custom game that to didnt work but did say cant find server

**Comment by dadobt** [comment]
the issue is resolved 
i guess it was some patch in [7.36c](https://liquipedia.net/dota2/7.36c) or [7.37](https://liquipedia.net/dota2/7.37)
